certified-connectors/Dynatrace/apiDefinition.swagger.json

```diff
@@ -0,0 +1,982 @@
+                  "type": "string",
+                  "description": "title"
+                },
+                "impactLevel": {
+                  "type": "string",
+                  "description": "impactLevel"
+                },
+                "severityLevel": {
+                  "type": "string",
+                  "description": "severityLevel"
+                },
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                },
+                "affectedEntities": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "entityId": {
+                        "type": "object",
+                        "properties": {
+                          "id": {
+                            "type": "string",
+                            "description": "id"
+                          },
+                          "type": {
+                            "type": "string",
+                            "description": "type"
+                          }
+                        },
+                        "description": "entityId"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "name"
+                      }
+                    }
+                  },
+                  "description": "affectedEntities"
+                },
+                "impactedEntities": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "entityId": {
+                        "type": "object",
+                        "properties": {
+                          "id": {
+                            "type": "string",
+                            "description": "id"
+                          },
+                          "type": {
+                            "type": "string",
+                            "description": "type"
+                          }
+                        },
+                        "description": "entityId"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "name"
+                      }
+                    }
+                  },
+                  "description": "impactedEntities"
+                },
+                "rootCauseEntity": {
+                  "type": "string",
+                  "description": "rootCauseEntity"
+                },
+                "managementZones": {
+                  "type": "array",
+                  "items": {},
+                  "description": "managementZones"
+                },
+                "entityTags": {
+                  "type": "array",
+                  "items": {},
+                  "description": "entityTags"
```

Comment:
@mandy-swinton please fix the description, it should be like `Entity tags`, but not in lowerCamelCase. Look through the others too please.

Reply:
I have updated all the descriptions to not be in camel case